pandas/core/generic.py

```diff
@@ -4694,6 +4702,10 @@ def add_suffix(self: NDFrameT, suffix: str) -> NDFrameT:
         ----------
         suffix : str
             The string to add after each label.
+        axis : {{0 or 'index', 1 or 'columns', None}}, default None
+            Axis to add suffix on
+
+             .. versionadded:: 1.5.0
```

Comment:
```suggestion
             .. versionadded:: 1.6.0
```

Reply:
thanks 😄, had missed this. 
updated this in both the changed methods docstring.